client/client.go

```diff
@@ -49,6 +52,15 @@ func newClient(opts Options) (*client, error) {
 	return &client{opts: opts, newSessionFn: newSession}, nil
 }
 
+func (c *client) Options() (AdminOptions, error) {
```

Comment:
I don't need to expose `Options` or `AdminOptions` if I change the method signatures as suggested in the other comment. I'd prefer to remove them to minimise changes. That good with you?